### ISSUE TITLE
index_date_pattern configuration does not take effect

### ISSUE THREAD

**Issue by ZMMWMY** [uken/fluent-plugin-elasticsearch#678]
(check apply)
- [x] read [the contribution guideline](https://github.com/uken/fluent-plugin-elasticsearch/blob/master/CONTRIBUTING.md)

#### Problem
According to the documentation, I set index_date_pattern to "" but got the error log.obviously, it didn't take effect.
btw the word``index_date_patter `` in the document is wrong
```
 #0 Error while index creation - <sxc-test.prod-{}-000001>: #<Elasticsearch::Transport::Transport::Errors::Forbidden: [403] {"error":{"root_cause":[{"type":"parse_exception","reason":"cannot parse empty date"}],"type":"security_exception","reason":"action [indices:admin/create] is unauthorized for user [elastic]","caused_by":{"type":"parse_exception","reason":"cannot parse empty date"}},"status":403}>
```
Configuration
```
<source>
  @type tail
  path /fluentd/test.txt
  <parse>
    @type none
  </parse>
  tag test
  @label @ERROR_LOG
</source>


<label @ERROR_LOG>

<match **>
  @type elasticsearch
  @log_level info
  log_es_400_reason true
  hosts 
  user 
  password 
# ILM configuration
  template_name test_template
  template_file /fluentd/template/test_template.json
  customize_template {"<<index_prefix>>": "docker"}
  rollover_index true
  deflector_alias test-alias
  index_prefix "sxc"
  application_name "test.prod"
  index_date_pattern ""
  retry_max_times 300
  reconnect_on_error true
  reload_on_failure true
  reload_connections false
  <buffer tag>
    @type file
    flush_thread_count 8
    path /fluentd/buffer/buffer-error-log/fluent.*.buffer
    flush_interval 6s
  </buffer>
</match>
</label>
```

**Comment by cosmo0920** [comment]
> According to the documentation, I set index_date_pattern to "" but got the error log.obviously, it didn't take effect.
> btw the word`index_date_patter ` in the document is wrong

This feature is un-released. Currently, only supported in master.

**Comment by ZMMWMY** [comment]
> > According to the documentation, I set index_date_pattern to "" but got the error log.obviously, it didn't take effect.
> > btw the word`index_date_patter ` in the document is wrong
> 
> This feature is un-released. Currently, only supported in master.

all right ,thanks

**Comment by cosmo0920** [comment]
I've released #674 as v3.7.1. Closing.
